src/websocket/WebsocketServer.js

```diff
@@ -409,6 +426,26 @@ module.exports = class WebsocketServer extends EventEmitter {
         }
     }
 
+    _pingConnections() {
+        const connections = [...this.connections.values()]
+        connections.forEach((connection) => {
+            try {
+                // didn't get "pong" in pingInterval
+                if (connection.isAlive !== undefined && !connection.isAlive) {
+                    throw Error('Connection is not active')
+                }
+
+                // eslint-disable-next-line no-param-reassign
+                connection.isAlive = false
```

Comment:
I'd consider renaming `isAlive` to something like `respondedPong` or `pongResponded`. It's quite easy to confuse `isAlive = false` to mean that the connection is actually dead right now.